### PR TITLE
adds reveal.nvim

### PR DESCRIPTION
**Pull request by haolian9**
hi, i made a plugin that let user use vifm inside nvim. could you add it into the TWiN?

### CODE REVIEW

File: contents/2022/Oct/24.md [hunk 1]
```diff
@@ -20,6 +20,22 @@
...
+    <span class="icon-text">
+      <span class="icon">
+        <i class="fa-solid fa-book"></i>
+      </span>
+      <span>reveal.nvim</span>
+    </span>
+  </a>
+</h3>
+
+reveal.nvim is a very experimental plugin that let user use vifm inside nvim through vifm's lua api.
```
Comment (phaazon) [inline contents/2022/Oct/24.md:34]:
```suggestion
`reveal.nvim` is a very experimental plugin that lets users use `vifm` inside Neovim through `vifm`'s lua API.
```

File: contents/2022/Oct/24.md [hunk 2]
```diff
@@ -20,6 +20,22 @@
...
+By [@haolian9].
```
Comment (phaazon) [inline contents/2022/Oct/24.md:35]: Please add the link at the bottom on the document. It doesn’t get generated automatically.

**Review by phaazon — CHANGES_REQUESTED** [review]

**Comment by haolian9** [comment]
hi phaazon, thanks for reviewing this. i just updated the text according to your suggests.

**Comment by haolian9** [comment]
phew, finelly got it mergeable

**Comment by phaazon** [comment]
Thanks!

**Review by phaazon — APPROVED** [review]

**Pull request merged**